docs/interactions/structured-messages/webview.md

```diff
@@ -0,0 +1,51 @@
+# Webview
+
+The webview is a convenient way of opening a custom made webpage. It can be chosen by setting the optional target field to `webview` when sending a Structured Message containing a link element, such as [Rich Link](../rich-link), [Carousel](../carousel), or [Template](../template).
+
+The webview is a small window that sits atop the chat. It can be customized in size through the `webview_height` parameter.
+
+## Use Case
+
+The webview is not just a simple window opening, it can be used to display a custom survey, to select a room or a seat on a train or even a more attractive way to select a subscription. For each action taken on the page, a custom response can be sent for the agent side, that can confirm a seat selection or express gratitude to a client indicating satisfaction.
+
+## Recommended Usage
+
+It should be noted that no default picker or page is provided and this feature is intended to be customizable, not out of the box. No default response after the customer has finished taking actions on the page is provided either.
+
+What is recommended is setting the conversation's `data-id` inside of the url query string in order to provide a custom page taylored to the customer (e.g displaying a confirmation instead of a choice the second times the customer clicks on it), avoiding him doing several times the procedure.
```

Comment:
```suggestion
What is recommended is setting the conversation's `data-id` inside of the url query string in order to provide a custom page taylored to the customer (e.g., displaying a confirmation instead of a choice the second time the customer clicks on it), avoiding him doing several times the procedure.
```

Reply:
Is the comma after `e.g` intended ?

Reply:
Yes, `e.g.` means `for example`. So here it's `For example, displaying a confirmation...`

Reply:
Nice I knew it meant `for example` in latin, but I did not know you you needed a comma after using it in an english sentence. I'll add it.